media/templates/homepage.mustache

```diff
@@ -1,236 +1,256 @@
-{{#>>classwork_table}}
-<div id="classwork_table">
-    <div>      
-        <div class="filter-widget left">
-            {{#>>switcher_collection_chooser}}
-            <div class="switcher_collection_chooser switcher collection-chooser" tabindex="1">            
-                <a class="switcher-top menuclosed" onclick="return switcher(event, this);">
-                    <span class="switcher-prefix">Author </span>
-                    <span class="title">{{{selected_label?ellipses(22)}}}</span>
-                    <span class="down-arrow" tabindex="3"></span>                
-                </a>
-                
-                <ul class="switcher-options">
-                    {{^showing_my_items}}
-                    <li>
-                        <li class="choice_my_items" id="choice_my_items">
-                            <a class="switcher-choice owner" href="{{space_viewer.username}}" style="font-weight: bold">Me</a>
+<div>
+  <div class="filter-widget left">
+    {{#switcher_collection_chooser}}
+      <div class="switcher_collection_chooser switcher collection-chooser" tabindex="1">
+        <a class="switcher-top menuclosed" onclick="return switcher(event, this);">
+          <span class="switcher-prefix">Author </span>
+          <span class="title">
+            {{#selected_label}}
+              {{{selected_label}}}
```

Comment:
Mustache allows you to do template functions like this by passing them into the context:
https://github.com/janl/mustache.js#functions

I'll add an ellipsis function so we retain this functionality.